src/modules/Builder/features/ActivityItems/ItemConfiguration/OptionalItemsAndSettings/OptionalItemsAndSettings.hooks.tsx

```diff
@@ -90,15 +91,16 @@ export const useSettingsSetup = ({
   handleAddOption,
   handleAddSliderRow,
   handleAddSingleOrMultipleRow,
-  removeAlert,
-  handleAddAlert,
+  // removeAlert,
```

Comment:
could you left a comment why they were hidden or might be they depend on some BE issue?